_posts/blog/2017-05-15-how-to-structure-product-feedback.md

```diff
@@ -0,0 +1,42 @@
+Providing feedback on products is tricky. When done properly it can serve as a valuable tool that fosters positive relationships and improves the products we build. When done poorly it can crush spirits and misinform the decisions we make. Through research and practice I’ve established several steps I now use to structure the way I give feedback. 
```

Comment:
Needs "front matter" at the top including categories and tags. Here's an example but you may want to include it in multiple categories, or change the tags. See other blog posts on the site for examples.

```yaml
---
title: How to Structure Product Feedback
author: Brandon Houlihan
categories: Product
tags: feedback team design
---
```